src/System.Diagnostics.PerformanceCounter/tests/PerformanceDataTests.cs

```diff
@@ -0,0 +1,257 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.PerformanceData;
+using System.IO;
+using Xunit;
+using Xunit.NetCore.Extensions;
+
+namespace System.Diagnostics.Tests
+{
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // In appcontainer, cannot write to perf counters
+    public class PerformanceDataTests : IClassFixture<PerformanceDataTestsFixture>
+    {
+        PerformanceDataTestsFixture _fixture = null;
+
+        public PerformanceDataTests(PerformanceDataTestsFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        /// <summary>
+        /// This test was taken from System.Diagnostics.PerformanceData documentation https://msdn.microsoft.com/en-us/library/system.diagnostics.performancedata(v=vs.110).aspx
+        /// To create provider.res file we've used some tools:
+        /// ctrpp.exe -legacy provider.man
+        /// rc.exe /r /i "c:\Program Files\Microsoft SDKs\Windows\v6.0\Include" provider.rc
+        /// </summary>
+        [ConditionalFact(typeof(Helpers), nameof(Helpers.IsElevatedAndCanWriteToPerfCounters))]
+        [Trait(XunitConstants.Category, XunitConstants.RequiresElevation)]
```

Comment:
https://github.com/dotnet/corefx/issues/30188#issuecomment-405623400 i think runs tests with admin privileges, @ericstj?

Reply:
It should filter tests to only run when the caller is elevated, at least that's what is supposed to happen, not sure of the current state.  https://github.com/dotnet/corefx/issues/17420

Reply:
@danmosemsft do you know if it's useful here?

Reply:
You can do something like this https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.EventLog/tests/Helpers.cs#L16
instead of using traits

Reply:
the nameof(Helpers.IsElevatedAndCanWriteToPerfCounters) condition should take care of the elevated requirement

Reply:
I cannot see any place that `[Trait(XunitConstants.RequiresElevation)] ` is wired up. I would use the `AdminHelpers.IsProcessElevated()` approach (such as @adiaaida says)

Reply:
i removed trait.